SongsApp/settings.py

```diff
@@ -138,3 +139,6 @@
 EMAIL_HOST_PASSWORD = config('EMAIL_HOST_PASSWORD')
 EMAIL_PORT = 587
 EMAIL_USE_TLS = True
+
+MEDIA_URL = '/media/'
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
```

Comment:
Always make sure to end a python file with a new empty line. Read on PEP-8 here (https://www.python.org/dev/peps/pep-0008/)